HowTos/SAML_Integration_PingOne_IdP.rst

```diff
@@ -0,0 +1,259 @@
+   | Entity ID                    | ``https://[host]/``                               |
+   +------------------------------+---------------------------------------------------+
+   | SLO endpoint                 | Not Specified                                     |
+   +------------------------------+---------------------------------------------------+
+   | SLO response endpoint        | Not Specified                                     |
+   +------------------------------+---------------------------------------------------+
+   | SLO binding                  | HTTP POST                                         |
+   +------------------------------+---------------------------------------------------+
+   | Assertion validity duration  | 300                                               |
+   +------------------------------+---------------------------------------------------+
+   | Target Application URL       | Not Specified                                     |
+   +------------------------------+---------------------------------------------------+
+   | Enforce signed Authn request | Disabled                                          |
+   +------------------------------+---------------------------------------------------+
+   | Verification certificate     | No Verification Certificates Selected             |
+   +------------------------------+---------------------------------------------------+
+
+   .. note::
+   
+      ``[host]`` is the hostname or IP of your Aviatrix controller.  For example, ``https://controller.demo.aviatrix.live``
+
+      ``[Endpoint Name]`` is an arbitrary identifier.  This same value should be used when configuring SAML in the Aviatrix controller.
+      
+      ``[Entity ID]`` is using ``https://[host]/`` as default if you select `Hostname` option when configuring SAML in the Aviatrix controller.
+      
+   |pingone_idp_configuring_saml_connection|
+      
+#. Click Save and Continue.
+
+#. For attribute mapping, click the button "+ADD ATTRIBUTE" and then select "PingOne Attribute" to map PingOne user attribute to an attribute in this application as below.
+
+   +------------------------+-----------------------+
+   | PINGONE USER ATTRIBUTE | APPLICATION ATTRIBUTE |
+   +------------------------+-----------------------+
+   | User ID                | saml_subject          |
+   +------------------------+-----------------------+
+   | Given Name             | FirstName             |
+   +------------------------+-----------------------+
+   | Family Name            | LastName              |
+   +------------------------+-----------------------+
+   | Email Address          | Email                 |
+   +------------------------+-----------------------+
+   
+   .. note::
+   
+      Notes: User ID is a default required in PingOne
+
+   ||pingone_idp_configuring_attribute_mapping||
+
+#. Click Save and Close.
+
+#. Enable the WEB SAML APP
+
+   |pingone_idp_enable|
+   
+.. _pingone_idp_metadata:
+
```

Comment:
In reference to https://docs.aviatrix.com/HowTos/SAML_Integration_Okta_IdP.html -- in the last step, it indicates that the application needs to be assigned to an account. 

Is there a need to do that here to complete (Step 2. Create a PingOne Web SAML App for Aviatrix)?

Reply:
PingOne for customer does not have that concept. Basically user and application are global as far as I know.